projects/plugins/jetpack/sal/class.json-api-site-jetpack.php

```diff
@@ -731,4 +731,15 @@ public function get_zendesk_site_meta() {
 	public function is_pending_plan() {
 		return false;
 	}
+
+	/**
+	 * Defaults to false instead of returning the hosting features for the current user on this site.
+	 *
+	 * @see class.json-api-site-jetpack-shadow.php on WordPress.com for implementation. Only applicable on WordPress.com.
+	 *
+	 * @return bool
+	 */
+	public function get_hosting_features() {
+		return false;
```

Comment:
Should this return `null` / empty array instead? The type is `array`.

Reply:
I followed the returned value of the get_plan function.